### ISSUE TITLE
Better styling for documentation pages

### ISSUE THREAD

**Issue by ogajduse** [spacewalkproject/spacewalkproject.github.io#11]
Especially API documentation is absolutely ugly - without any styling, just pure HTML.
There should be some CSS file which will improve the design of the page.

**Comment by jiridostal** [comment]
You're right, but I don't see this a a big problem. A lot of API documentations online look very similar and I didn't even notice there's something missing... But we may create some quick CSS
